example/ViewPagerExample.js

```diff
@@ -35,6 +35,7 @@ type State = {
   },
   pages: Array<CreatePage>,
   scrollState: PageScrollState,
+  areDotsVisible: boolean,
```

Comment:
This name does not follow the convention we use int the rest of the project. Please use `dotsVisible` or just `showPageIndicator`